db/__tests__/seed.test.js

```diff
@@ -333,4 +321,31 @@ test('DELETE 400: responds with error message when comment_id is invalid', () =>
     .delete('/api/comments/fdsa')
     .expect(400)
 })
+})
+describe.only('/api/users', () => {
+test('GET 200: responds with array of users', () => {
+    return request(app)
+    .get('/api/users')
+    .expect(200)
+    .then(({ body }) => {
+    const { users } = body
+    users.forEach((user) => {
+        expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String)
+        }
+        )
+    })
+})
+});
+test('GET 404: responds with bad request if endpoint is invalid', () => {
```

Comment:
No need to test for this bad path endpoint inside this describe block and the functionality for this endpoint. This logic doesn't touch this endpoints controller or model and you have probably already covered all bad paths, did this auto pass?